Source/ProgressionSystemRuntime/Public/Data/PSWorldSubsystem.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+#pragma once
+
+#include "Subsystems/WorldSubsystem.h"
+#include "PSWorldSubsystem.generated.h"
+
+/**
+ * Implements the world subsystem to access different components in the module 
+ */
+UCLASS(BlueprintType, Blueprintable, Config = "ProgressionSystem", DefaultConfig)
+class PROGRESSIONSYSTEMRUNTIME_API UPSWorldSubsystem : public UWorldSubsystem
+{
+	GENERATED_BODY()
+
+public:
+	/** Returns this Subsystem, is checked and wil crash if can't be obtained.*/
+	static UPSWorldSubsystem& Get();
+	static UPSWorldSubsystem& Get(const UObject& WorldContextObject);
+
+	/** Returns the data asset that contains all the assets of Progression System game feature.
+	 * @see UPSWorldSubsystem::PSDataAssetInternal. */
+	UFUNCTION(BlueprintCallable, BlueprintPure, Category = "C++")
+	const class UPSDataAsset* GetPSDataAsset() const;
+	
+	/** Returns a progression System component reference */
+	UFUNCTION(BlueprintPure, Category = "C++")
+	FORCEINLINE class UPSHUDComponent* GetProgressionSystemComponent() const { return ProgressionSystemComponentInternal;	}
+
+	UFUNCTION(BlueprintCallable, Category = "C++")
+	void SetProgressionSystemComponent(UPSHUDComponent* MyProgressionSystemComponent);
+
+protected:
+	/** Contains all the assets and tweaks of New Main Menu game feature.
```

Comment:
wrong `New Main Menu` copypaste

Reply:
it's still `BasePSDataAssetInternal.ini` instead of `BaseProgressionSystem.ini`

Reply:
![image](https://github.com/h4rdmol/ProgressionSystem/assets/5227233/992ac952-1189-41d7-826f-1c5ffff574a2)
can't reproduce. Is it somehow cached in rider or ignored?

Reply:
In logic is everything is fine, there is no issue with that, it's just wrong copypasta, just search in your code comments for BasePSDataAssetInternal.ini